### PR TITLE
Adding :strawberry: to known CakePHP 4.x compatible

### PR DESCRIPTION
**Pull request by challgren**
https://github.com/jomweb/CakeImpersonate/releases/tag/4.0.0-beta
https://github.com/narendravaghela/cakephp-mailgun/releases/tag/5.0.0-beta
https://github.com/bcrowe/cakephp-encrypted-type/tree/cake-4.x

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -245,7 +245,7 @@ Additional lists you might find useful:
 - [CakeAdmin plugin](https://github.com/cakemanager/cakephp-cakeadmin) - A non-stable user management plugin with a built-in admin area.
 - [CakeDC/Enum plugin](https://github.com/CakeDC/enum) - A plugin to add enumeration list support to your app.
 - :strawberry: [CakeDto plugin](https://github.com/dereuromark/cakephp-dto) - Quickly generate useful data transfer objects for your app (mutable/immutable), replacing messy arrays and leveraging your IDE through typehinting and autocomplete.
-- [CakeImpersonate plugin](https://github.com/jomweb/CakeImpersonate) - A component that stores the current authentication session and creates new session for impersonating Users. User can revert back to original authentication sessions without the need to re-login.
+- :strawberry: [CakeImpersonate plugin](https://github.com/jomweb/CakeImpersonate) - A component that stores the current authentication session and creates new session for impersonating Users. User can revert back to original authentication sessions without the need to re-login.
```
Comment (ADmad) [inline README.md]: I don't see how this plugin's `4.0.0-beta` release is Cake 4 compatible when it's composer.json has this https://github.com/jomweb/CakeImpersonate/blob/4.0.0-beta/composer.json#L8

File: README.md [hunk 2]
```diff
@@ -341,7 +341,7 @@ Additional lists you might find useful:
 ## Security
 *Plugins and information around security, preventing vulnerabilities and protection against XSS and alike.*
 
-- [BryanCrowe/EncryptedType](https://github.com/bcrowe/cakephp-encrypted-type) - Simple solution that enables Encrypted data types to be stored in the database.
+- :strawberry: [BryanCrowe/EncryptedType](https://github.com/bcrowe/cakephp-encrypted-type) - Simple solution that enables Encrypted data types to be stored in the database.
```
Comment (ADmad) [inline README.md]: This plugin doesn't have any Cake 4 compatible release, just a branch. At least a beta release is necessary to earn the strawberry icon.

File: README.md (hunk 1 above)
Comment (challgren) [reply README.md]: Ahh good catch! Must have tagged the wrong branch. But https://github.com/jomweb/CakeImpersonate/releases/tag/4.0.0-beta1 is now available with the correct tagged branch.

File: README.md (hunk 2 above)
Comment (challgren) [reply README.md]: @bcrowe said he was going to do a release.

Comment (challgren) [reply README.md]: I'll split up the PR.

**Pull request closed without merging**